pyslackers_website/slack/tasks.py

```diff
@@ -55,14 +57,25 @@ def capture_snapshot_of_user_count() -> None:
     slack = SlackClient(settings.SLACK_OAUTH_TOKEN)
 
     member_count = deleted_count = bot_count = 0
+    time_zones = []
+
     for member in slack.members():
         if member.get('is_bot'):
             bot_count += 1
         elif member.get('deleted'):
             deleted_count += 1
         else:
             member_count += 1
+            tz = member.get('tz')
+            if tz is not None:    
+                time_zones.append(tz)
+    
+    counter = Counter(time_zones)
+    tz_count_json = dumps(dict(counter))
```

Comment:
Nice usage of counter, however I would prefer to not manually dump the json and let the orm handle it. https://docs.djangoproject.com/en/1.11/ref/contrib/postgres/fields/#jsonfield

Secondly, from what I recall with the slack API and the Django cache - it should work with the object you're caching since it's a non-complex dict

Reply:
(continued, since reviews on mobile are : 💩)

For item 2, any reason to manually dump vs allowing the cache abstraction to deal with it? One favorable reason is json vs pickle, but the context and cache accept Python types so I favor that